pkg/byteorder/byteorder.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 Kinvolk GmbH
```

Comment:
Shouldn't the copyright be from Weaveworks? In fact, we should probably not have any headers. (cc @rade )